week_3/README.md

```diff
@@ -0,0 +1,10 @@
+HomeWork3
```

Comment:
You shouldn't add this file.

Reply:
I fixed that. Hope everyone can follow the rules